core/vm/jump_table.go

```diff
@@ -499,6 +499,12 @@ func newFrontierInstructionSet() InstructionSet {
 			validateStack: makeStackFunc(0, 0),
 			valid:         true,
 		},
+		PUSH0: {
```

Comment:
You also added INVALID opcode. Shouldn't it be also added somewhere in this file?